git-reader/gitreader/src/logger.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017-2021 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import time
+import sys
+
+DATE_FORMAT = "%Y-%m-%d %H:%M:%S.%f"
+
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+class BrowserLogger(object):
+
+    def __init__(self, log_dir=None):
```

Comment:
It would be nice to configure logs_dir based on envvars/argument - to store logs in mounted folder